.blinkmrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "server": {
+    "project": "bm-service-settings.blinkm.io",
+    "region": "ap-southeast-2"
```

Comment:
I think its ok. It we want to get to the type of project registry you suggested during the email chain re. tenants. We might be better to standardise on how we stored a project identifier so our CLIs can use the same library /shrug